apps/store/themes/store/js/list_assets.js

```diff
@@ -310,59 +518,64 @@ $(window).load(function () {
         });
 
         // bellow code block will generate sub categories for only
-        var nodes = getURL().split("/");
-        var path = nodes[0];
+        if (assetAvailability) {
+            var nodes = getURL().split("/");
+
+            var path = nodes[0];
+
+            for (var i = 1; i < nodes.length; i++) {
+                path += "/" + nodes[i];
+                var elementRef = document.getElementById(path);
```

Comment:
Since "/" contain the path variable JQuery does not support selection with "/" symbol. As i remember with windows IE